lib/Data/Validate/Sanctions.pm

```diff
@@ -112,37 +211,37 @@ sub get_sanctioned_info {    ## no critic (RequireArgUnpacking)
         my @names = keys %{$data->{$file}->{names_list}};
 
         foreach my $sanctioned_name (sort @names) {
+            my $entry = $data->{$file}->{names_list}->{$sanctioned_name};
```

Comment:
Is it possible to have multiple entries in the same sanction list with the same name? if so we should not merge them.

Reply:
OK. It will be fixed.

Reply:
Entries are saved in an array with no merging. to simplify lookups, an index is introduced.